CMakeLists.txt

```diff
@@ -14,8 +14,7 @@
 #------------------------------------------------------------------------------#
 cmake_minimum_required(VERSION 3.16)
 
-project(Cabana LANGUAGES CXX)
-set(PROJECT_VERSION "1.0-dev")
+project(Cabana LANGUAGES CXX VERSION 0.6.0)
```

Comment:
```suggestion
project(Cabana LANGUAGES CXX VERSION 0.7.0-dev)
```

Reply:
Makes sense, but apparently `project` doesn't allow non-numeric versions. I applied your suggestion to the previous version command